packages/ui/src/components/Icon/icons/index.tsx

```diff
@@ -86,6 +86,7 @@ export * as IconCrosshair from './IconCrosshair'
 export * as IconDatabase from './IconDatabase'
 export * as IconDelete from './IconDelete'
 export * as IconDisc from './IconDisc'
+export * as IconDiscussions from './IconDiscussions'
```

Comment:
@MildTomato it probably makes sense for us to extend this existing Icons component with custom ones so we can continue using a standardized api.